timer/api/timer.go

```diff
@@ -15,18 +15,26 @@
 package api
 
 import (
+	"strings"
 	"time"
 
 	"github.com/pingcap/errors"
 	"github.com/pingcap/tidb/parser/duration"
+	"github.com/pingcap/tidb/util/timeutil"
+	"github.com/robfig/cron/v3"
 )
 
+// TimeZoneTiDB indicates to use the tidb timezone.
```

Comment:
It seems that `TimeZoneTiDB` is the same with an empty string for 'timezone' fields in timer. IMO its semantic is not better than an empty string to me -_-.

How about just removing it and explicitly explain that in the client api? For example we can declare that 'if timezone is empty string, we use the (global) `time_zone` system variable'

Reply:
In the previous implement of timer, it uses `TiDB` as the default value and has already persisted into `mysql.tidb_timers`. 

But I think we can just support parsing `TiDB` value without  supporting write `TiDB` for compatitability